crates/ruff_linter/src/rules/flake8_logging_format/violations.rs

```diff
@@ -432,7 +432,7 @@ impl AlwaysFixableViolation for LoggingWarn {
 ///
 /// username = "Maria"
 ///
-/// logging.info("Something happened", extra=dict(user=username))
+/// logging.info("Something happened", extra=dict(user_id=username))
```

Comment:
Should we instead adjust the `basicConfig` call above so that the two examples are consistent?

Reply:
The other rules also use `user_id`, so I believe this is the consistent choice. The variable name `username` is on purpose different from the keyword to make the relation between variable and log formatting clear.